docs/docs-content/user-management/palette-resource-limits.md

```diff
@@ -25,7 +25,7 @@ The following table lists the default resource limits for Palette:
 | Workspaces        | 50            | Tenant    |
 | Roles             | 100           | Tenant    |
 | Cloud Accounts    | 200           | Tenant    |
-| Cluster Profiles  | 200           | Tenant    |
+| Cluster Profiles (profile versions count as well) | 200           | Tenant    |
```

Comment:
@yuliiiah, do you have thoughts/ideas on how we can better represent this versus using the parenthesis?